Cargo.toml

```diff
@@ -3,6 +3,4 @@ members = [
   "rustler",
   "rustler_codegen",
   "rustler_sys",
-  "rustler_tests/native/rustler_test",
```

Comment:
I think I would not drop this. With this, running `cargo check` or similar in the root directory will also check the test crates, without having to run `mix test` in `rustler_tests` or `cargo check` for each of the test crates. For example, if something in `rustler_codegen` changes, problems are likely to be detected when running `cargo check` for `rustler_test`.

Reply:
Makes sense :+1: